sensor:
  serial_num: "D20928"
  resolution: "QVGA"
  fps: 30
  T_cam_offset:
    - [2.22e-16, 2.22e-16, -1.00e+00, 0.00e+00]
    - [-1.00e+00, 0.00e+00, -2.22e-16, 0.00e+00]
    - [0.00e+00, 1.00e+00, 2.22e-16, 1.50e-02]
    - [0.00e+00, 0.00e+00, 0.00e+00, 1.00e+00]

  P:
    - [2.30940108, 0.0, 0.0, 0.0]
    - [0.0, 1.73205081, 0.0, 0.0]
    - [0.0, 0.0, -1.04081633, -0.00204081633]
    - [0.0, 0.0, -1.0, 0.0]

  z_near: 0.001
  z_far: 0.05
  gel_width: 0.01835
  gel_height: 0.02460
  gel_thickness: 0.0046
  gel_min_depth: 0.01954

mm_to_pixel: 17.15
ball_diameter: 6.0
max_depth: 0.02076

visualize:
  normals: True
  points3d: True
  ellipse: True

dataloader:
  batch_size: 1
  shuffle: False
  num_workers: 8
  annot_flag: True
  annot_file: "annotate.csv"

dataset:
  dataset_type: "imgs"
  save_dataset: True
  save_depth: True
